verification_service/main.py

```diff
@@ -164,22 +163,20 @@ def process_event(self, queue_name, data):
         user_verified = False
 
         # Determine how to get the user email address based on event type
-        if (
-            event_type
-            in project_events + user_events + issue_events + issue_note_events
-        ):
+        if event_type in [e.value for e in ProjectEvent] \
+        or event_type in [e.value for e in UserEvent] \
+        or event_type in [e.value for e in IssueEvent] \
+        or event_type in [e.value for e in IssueNoteEvent]:
```

Comment:
**nitpick:** I'd wrap it in () instead of escaping the newline, makes code formatters have an easier time formatting things.